spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
-kind,scalerank
+kind,operator,zoom::int,area::int,output:tier::int,output:scalerank::int
+national_park;park or protected land;battlefield,!United States Forest Service,>=3,>=300000000,1,3
+national_park;park or protected land;battlefield,!United States Forest Service,>=4,>=300000000,1,4
+national_park;park or protected land;battlefield,!United States Forest Service,>=5,>=150000000,1,5
+national_park;park or protected land;battlefield,!United States Forest Service,>=6,>=150000000,1,6
+national_park;park or protected land;battlefield,!United States Forest Service,>=7,>=100000000,1,7
+national_park;park or protected land;battlefield,!United States Forest Service,>=8,>=10000000,1,8
+national_park;park or protected land;battlefield,!United States Forest Service,>=9,>=5000000,1,9
+national_park;park or protected land;battlefield,!United States Forest Service,>=10,>=1000000,1,10
+national_park;park or protected land;battlefield,!United States Forest Service,>=11,>=500000,1,11
+national_park;park or protected land;battlefield,!United States Forest Service,>=12,>=500000,1,12
+national_park;park or protected land;battlefield,!United States Forest Service,>=13,>=100000,1,13
+national_park;park or protected land;battlefield,!United States Forest Service,>=14,>=50000,1,14
+national_park;park or protected land;battlefield,!United States Forest Service,>=15,>=2000,1,15
+national_park;park or protected land;battlefield,!United States Forest Service,>=16,+,1,16
+national_park;park or protected land,!United States National Park Service,>=4,>=1000000000,2,4
+national_park;park or protected land,!United States National Park Service,>=5,>=1000000000,2,5
+national_park;park or protected land,!United States National Park Service,>=6,>=150000000,2,6
+national_park;park or protected land,!United States National Park Service,>=7,>=100000000,2,7
+national_park;park or protected land,!United States National Park Service,>=8,>=10000000,2,8
+national_park;park or protected land,!United States National Park Service,>=9,>=5000000,2,9
+national_park;park or protected land,!United States National Park Service,>=10,>=1000000,2,10
+national_park;park or protected land,!United States National Park Service,>=11,>=500000,2,11
+national_park;park or protected land,!United States National Park Service,>=12,>=250000,2,12
+national_park;park or protected land,!United States National Park Service,>=13,>=100000,2,13
+national_park;park or protected land,!United States National Park Service,>=14,>=50000,2,14
+national_park;park or protected land,!United States National Park Service,>=15,>=2000,2,15
+national_park;park or protected land,!United States National Park Service,>=16,+,2,16
+conservation;protected_area;nature_reserve;urban;rural;residential,,>=4,>=1000000000,2,4
+conservation;protected_area;nature_reserve;urban;rural;residential,,>=5,>=1000000000,2,5
+conservation;protected_area;nature_reserve;urban;rural;residential;forest;wood,,>=6,>=150000000,2,6
```

Comment:
- **red** = tier 2 kind IN (conservation, protected_area, nature_reserve) && protect_class IN ("6",6) && $zoom: { min: 4 } _DOES SHOW UP_
  - https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L6236-L6241
- **pink** - tier 2 kind IN (park, national_park) && operator NOT "National Park Service" && $zoom: { min: 4 } _DOES SHOW UP_
  - https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L6284-L6291
  - Outside of the USA these actually look to be true national parks
- **blue** = tier 2 kind IN (forest, natural_forest, wood, natural_wood, natural_park) && operator IN (United States Forest Service) && $zoom: { min: 6 } _NEVER SHOWS UP_
  - https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L6253
  - See yellow section below for continuation.
- **yellow** = tier 2 kind IN (forest, natural_forest, wood, natural_wood, natural_park) && $zoom: { min: 10 } && operator NOT "United States Forest Service" _DOES SHOW UP_
  - https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L6262-L6269
  - Note that we refactored kinds in v1-pre1 so `natural_forest, natural_wood, natural_park` need to be added to the filter starting at zoom 10.
- **purple** - tier 2 kind IN (natural_forest, natural_wood, natural_park) && operator NOT "United States Forest Service" && $zoom: { min: 10 } _DOES SHOW UP_
  - _Only on my local machine as a modification over the pre1 branch of Bubble Wrap_

![us_national_forests2](https://cloud.githubusercontent.com/assets/853051/17919062/426ac9f4-697f-11e6-85d3-acb0246a71fa.gif)